include/z64frameadvance.h

```diff
@@ -0,0 +1,19 @@
+#ifndef Z64FRAMEADVANCE_H
+#define Z64FRAMEADVANCE_H
```

Comment:
in OoT we have z_frame_advance.c (in MM it's z_pause.c), which suggests that this should be named z64frame_advance.h